webhook-bridge/svix-webhook-bridge.example5.receivers.yaml

```diff
@@ -0,0 +1,65 @@
+log_level: "debug"
```

Comment:
Everything looks excellent in this and the senders examples!

One thing I'd make sure that we support in this YAML is loading from env var. Does YAML by any chance have an extension that supports just loading from env vars so I can do? `key: $MY_KEY` in the config and then we'll load the key from env var or env var file?

Reply:
Nothing is provided for this in `serde_yaml` from what I saw. We could pre-process the config to expand vars before parsing the YAML using something like [shellexpand](https://crates.io/crates/shellexpand).

In the past I'd used [envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html), in a shell command but that's a bit clumsy for launching services. My use case was for deployments, etc where you're usually running a series of commands anyway.